src/app/components/client/ExposureCard.tsx

```diff
@@ -132,50 +149,68 @@ export const ExposureCard = (props: Props) => {
             detailsOpen ? styles.isOpen : ""
           }`}
         >
-          <p>
-            This site is selling and publishing{" "}
-            <a href={exposureDetailsLink}>
-              details about you.{" "}
-              <span>
-                <OpenInNew alt="" width="15" height="14" />
-              </span>
-            </a>{" "}
-            Remove this profile to protect your privacy.
-          </p>
+          {isScanResult(exposureData) ? 
+          // Data broker content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-info-for-sale-part-one", 
+              {
+                elems: {"data_broker_link": <a href={exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt="" width="13" height="13" aria-label={l10n.getString("open-in-new-tab-aria-label")} />
+                </span>
+              </a>
+              {" "}{l10n.getString("exposure-card-description-info-for-sale-part-two")}
+            </p>
+          </div> : 
+          // Data breach content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-data-breach-part-one", 
+              {
+                vars: {
+                  data_breach_company: exposureName,
+                  data_breach_date: exposureData.BreachDate
```

Comment:
Q: Is this date localized somewhere, or is it the raw value from HIBP?

If they aren't localized, what date is 07/10/2020?

<details>
<summary>Spoiler alert</summary>

```js
const d = new Date("07/10/2020");
console.log(d.toString()); // 'Fri Jul 10 2020 00:00:00 GMT-0700 (Pacific Daylight Time)'
console.log("LOCALE | DATE\n:----:|----");
for (const locale of locales) {
  console.log(`${locale} | ${ new Intl.DateTimeFormat(locale).format(d)}`);
}
```

LOCALE | DATE
:----:|----
be | 10.7.2020
cs | 10. 7. 2020
cy | 10/7/2020
da | 10.7.2020
de | 10.7.2020
dsb | 10.7.2020
el | 10/7/2020
en-CA | 7/10/2020
en-GB | 10/07/2020
es-AR | 10/7/2020
es-CL | 10-07-2020
es-ES | 10/7/2020
es-MX | 10/7/2020
fi | 10.7.2020
fr | 10/07/2020
fy-NL | 10-7-2020
gn | 7/10/2020
he | 10.7.2020
hsb | 10.7.2020
hu | 2020. 07. 10.
ia | 10-07-2020
is | 10.7.2020
it | 10/7/2020
ja | 2020/7/10
ka | 10.7.2020
kab | 10/7/2020
kk | 10.07.2020
ko | 2020. 7. 10.
nl | 10-7-2020
nn-NO | 10.7.2020
pa-IN | 10/7/2020
pl | 10.07.2020
pt-BR | 10/07/2020
pt-PT | 10/07/2020
rm | 10-07-2020
ru | 10.07.2020
sk | 10. 7. 2020
sl | 10. 7. 2020
sq | 10.7.2020
sr | 10. 7. 2020.
sv-SE | 2020-07-10
tr | 10.07.2020
uk | 10.07.2020
vi | 10/7/2020
zh-CN | 2020/7/10
zh-TW | 2020/7/10

```